### PR TITLE
Wrap empty inline XBRL elements

### LINKED ISSUE #1015: [REQUEST] highlighting for facts with xsi:nil="true"
### What should we change and why?

As more and more European filings seem to use `xsi:nil="true"` (probably accoring to the [ESEF reporting manual](https://www.esma.europa.eu/sites/default/files/library/esma32-60-254_esef_reporting_manual.pdf), Guidance 2.2.5), the iXBRL viewer should probably show these facts a bit more prominent.

E.g. if you look at this [filing](https://filings.xbrl.org/52990006IVXY7GCSSR39/2025-12-31/ESEF/FR/0/sartoriusstedimbio-2025-12-31-1-fr/reports/ixbrlviewer.html#f-fact-354) (fact 354) the NIL tagged value is almost invisible. To me it seems the viewer does not highlight the `td` (like for other facts) but creates an additional `<span>`, which spans "nothing" and due to that is not clickable nor highlighted.

Thanks and best regards,
Sacha

### PR DESCRIPTION
**Pull request by austinmatherne-wk**
#### Reason for change

Nil iXBRL fact values cannot currently be highlighted or selected in the DOM.

| master | branch |
| -- | -- |
| master  |  branch |

#### Description of change

Wrap empty elements, using the table cell when appropriate. Add unit tests for empty and nested elements.

#### Steps to Test

* CI

**review**:
@Arelle/arelle
@paulwarren-wk

### CODE REVIEW

**Review by brettkail-wk — DISMISSED** [review]

File: iXBRLViewerPlugin/viewer/src/js/viewer.js [hunk 1]
```diff
@@ -283,7 +288,7 @@ export class Viewer {
         const tableNode = domNode.closest("td,th");
         let nodes;
         const innerText = $(domNode).text();
-        if (tableNode !== null && getComputedStyle(tableNode).display === 'table-cell' && innerText.length > 0) {
+        if (tableNode !== null && getComputedStyle(tableNode).display === 'table-cell') {
```
Comment (brettkail-wk) [inline iXBRLViewerPlugin/viewer/src/js/viewer.js:294]: I'm not entirely sure why commit 34abe4578fb6bfe8757b641d6163c7bcbaf81d0f added that `innerText.length > 0`, but the code appears to work correctly for empty string.

Comment (paulwarren-wk) [reply iXBRLViewerPlugin/viewer/src/js/viewer.js:294]: It's not clear to me either. I suspect that it was to avoid calling `.indexOf("")` but behavior of this is well-defined, and the code does what we want in this case.

**Review by brettkail-wk — APPROVED** [review]

**Review by aaroncameron-wk — APPROVED** [review]
